index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://kit.fontawesome.com/5d13e94c8b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="nav">
+        <h1>Kayonga Chiteta</h1>   
+        <i class="fas fa-bars two"></i>
+    </nav>
+    <div class="container">
+        <h2>Hey There.<br>I'm Kayonga</h2>
+        <h3>I’am a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <ul class="list">
+                <li><a href="#"><i class="fa fa-github"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium"></i></a></li>
+            </ul>
+    </div>
```

Comment:
- [ ] You can leverage the `section` semantic element to wrap this section instead of the generic `div` element. Making use of semantic elements enhances Search Engine optimization and helps the browser to identify sections of your web app correctly.

- [ ] Please, it would be better to use an `h1` for the heading in line 14 instead of `h2`. It is always a good practice to have one and only one h1 element on a web page.